src/locales/zh/translation.json

```diff
@@ -82,7 +82,7 @@
   "Close modal": "关闭模态框",
   "Debug": "调试",
   "Options": "选项",
-  "<0>Open in OSM</0> — Opens the current view on openstreetmap.org": "在 openstreetmap.org 打开当前视图",
+  "<0>Open in OSM</0> &mdash; Opens the current view on openstreetmap.org": "在 openstreetmap.org 打开当前视图",
```

Comment:
This looks duplicated...?

Reply:
Yeah I'm not sure what's up with that. The i18n tool generated it but I can only find one string in the project.

Reply:
Correct.

Are there any instructions on how to test languages? Strangely, I have set `Accept-Language: zh` in my browser to try testing this, but the entire UI is still English, so I'm not sure the best way to verify if we can drop one string.

Reply:
You can use the dropdown to change languages...
Note that the current language is stored in local storage somewhere, so you need to be in incognito mode to see the language from the browser settings...

Reply:
oh heh... that's actually somewhat surprising 😅 Turns out the — rather than the HTML entity is the correct one ;)

Current maplibre.org/maputnik Japanese debug UI:

<img width="438" alt="image" src="https://github.com/user-attachments/assets/14ccb931-90ee-4221-88d9-5ee964c17346">

Local version after changing the Japanese translation to use the UTF-8 literal:

<img width="423" alt="image" src="https://github.com/user-attachments/assets/e10d5b69-ccfc-4d85-9040-4ae888a5c0b0">

Reply:
Thanks!